firecrown/ccl/two_point.py

```diff
@@ -28,10 +28,14 @@ def parse_config(analysis):
     if 'likelihood' in analysis:
         new_keys['likelihood'] = _parse_likelihood(analysis['likelihood'])
 
-    # read data if there is a sacc file
-    if 'sacc_file' in analysis:
+    # read data if there is a sacc file or read object
+    sacc_data = None
+    if 'sacc_data' in analysis:
+        sacc_data = analysis['sacc_data']
+    elif 'sacc_file' in analysis :
```

Comment:
We should use a single keyword here called `sacc_data` that is either the object or a string that gets read into a file. There is no reason to have two and that will make things simpler.

Reply:
Fixed that. Kept sacc_file with deprecation warning.